### ISSUE TITLE
zoneparser and include

### ISSUE THREAD

**Issue by miekg** [miekg/dns#798]
Seems the new zone parser is different and I'm either failing to use the new API or we have a regression.

~~~
=== RUN   TestInclude
--- FAIL: TestInclude (0.00s)
    include_test.go:30: Failed to find "foo.example.org." in parsed zone
~~~
https://travis-ci.org/coredns/coredns/jobs/444133894

**Comment by miekg** [comment]
ah:
~~~
// The directives $INCLUDE, $ORIGIN, $TTL and $GENERATE are all
// supported. Although $INCLUDE is disabled by default.
~~~

**Comment by miekg** [comment]
$INCLUDE is specified in 1035 and should on by default. Having the option to disable it is *very* nice to have, but the default should be on.

**Comment by miekg** [comment]
[ Quoting <notifications@github.com> in "Re: [miekg/dns] zoneparser and incl..." ]
>Like I believe the API should be fail-safe not fail-open. I’d rather someone wonder “Why aren’t the $INCLUDE directives working?” than having no idea and allowing unprivileged users to read root protected files for example.

yes, but it's specified in the standard as being supported, so default must be 
'on'. If you have untrusted zone files on disk you may have other problems.

Restricting where to load from is a nice feature to have though.

**Comment by tmthrgd** [comment]
@miekg You closed #799 after https://github.com/miekg/dns/pull/799#issuecomment-432091758, are you content that this is opt-in approach is okay?
